anime_downloader/commands/watch.py

```diff
@@ -67,7 +67,7 @@ def command(anime_name, new, update_all, _list, quality, remove,
         else:
             query = click.prompt('Enter a anime name or url', type=str)
 
-        url = util.search(query, provider)
+        url, _ = util.search(query, provider)
```

Comment:
does this cause any unintended behaviour that you have noticed?

Reply:
No, I literally see no reasons why this would cause any problems

Reply:
Yeah, it doesn't. This was commented before testing.. just wanted to be made aware